components/molecule/collapsible/demo/ArticleNoCollapse.js

```diff
@@ -36,6 +36,52 @@ const ArticleNoCollapse = ({className, icon, showText, hideText, height}) => {
           </div>
         </div>
       </DemoWrapper>
+      <Paragraph>
+        The component is set to not collapse once expanded, regardless of
+        contents' height
+      </Paragraph>
+      <br />
+      <br />
+      <DemoWrapper>
+        <div style={{display: 'flex', flexDirection: 'column'}}>
+          <span>Contents can not be collapsed, once expanded</span>
+          <div
+            style={{
+              backgroundColor: '#fff',
+              fontSize: 14,
+              padding: 16,
+              textAlign: 'left',
+              maxWidth: 400
+            }}
+          >
+            <MoleculeCollapsible
+              icon={icon}
+              isCollapsible={false}
```

Comment:
It is little strange to me that a MoleculeCollapsible component may not be collapsible. 🧐

Reply:
good point... perhaps it makes more sense to have something like this rather than, IDK, another component say `molecule/expandable` for cases where we only want the user to be able to expand the contents, and not collapse them back.

In the end, the same could be argued about the `disabled` prop for a button and yet, I still want to have that `prop` 😉